k8s/cloud/public/base/artifact_tracker_versions.yaml

```diff
@@ -1,3 +1,5 @@
+# yamllint disable-file
```

Comment:
can you just disable the indentation rule? so that we catch other errors in this file (if any)
ideally we just disable the rule in the affected section too (for example see places we disable the line-length rule in yamls)

Reply:
Sounds good, updated in 6ffcb58. I also needed to disable the `comments` rule since `yq` was removing the comment's second leading space. Rather than having two "yamllint disable" comment strings (`disable-line` for comments and `disable` for indentation), I opted to condense it to a single line.

Reply:
IMO just change the comment rule to make it easier for automated updates. Add the following rule to the yamllint.rc
```
comments:
    min-spaces-from-content: 1
```

This ensures that if you use an automated tool like rennovatebot/dependabot to update GH actions, it also passes the linter.

Reply:
Good call. Updated in latest revision.